scenes/game/local_game.gd

```diff
@@ -6130,6 +6225,12 @@ func boost_finish_resolving_card(performing_player : Player):
 	return events
 
 func boost_play_cleanup(events, performing_player : Player):
+	# Account for boosts that played other boosts
+	if active_boost.parent_boost:
+		active_boost = active_boost.parent_boost
```

Comment:
I think there's probably something about if this boost was played mid-strike it needs to add the boost effects to the active effects, but let's wait until we have a character that plays boosts mid-strike that also play boosts...
This is complicated enough as is.

Reply:
can add in an assertion as a reminder, in case that happens